components/gitpod-protocol/src/messaging/error.ts

```diff
@@ -119,6 +119,9 @@ export const ErrorCodes = {
     // 501 EE Feature
     EE_FEATURE: 501 as const,
 
+    // 501 Unimplemented
+    UNIMPLEMENTED: 501 as const,
```

Comment:
it should be different code, EE_FEATURE is 501?

Reply:
or delete EE_FEATURE if it is not used anymore.

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/501

Reply:
it does not matter there error codes are not HTTP codes

Reply:
ok, will update later with unit tests

Reply:
Addressed in a pending commit, not going to remove EE_FEATURE just in case we have other clients are using it. (Even we thought it shouldn't, right?)